funding_csv_to_hash.rb

```diff
@@ -0,0 +1,45 @@
+require 'csv'
+
+funding_csv = CSV.read("prod_funding_sources.csv")
+funding_csv.shift
+formatted_funding_data = funding_csv.each_with_object({}) do |row, hash|
+  # name, ror-identifier, award-number, file-id
+  name = row[0]
+  identifier = row[1]
+  award_number = row[2]
+  file_id = row[3]
+
+  if hash.keys.include?(file_id)
+    hash[file_id] +=
```

Comment:
I'm not sure if there is a better way to do this, but what's happening with this if/else is that if there is no key (file_id) in the hash, it will fail the first statement and then create an array with the funding data. If there is a key (file_id) in the hash, it will add to the array that's already there. I had an annoying bug with this were I was just overwriting the entries over and over when I needed to add to it for files with multiple funding sources.